### DISCUSSION TITLE
We have a LinkFree Profile!

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#25] [Team Posts]
###### Join our org [Here ✉️](https://github.com/App-Choreography/Get-An-Invite/issues/new?assignees=CodingSpecies&labels=Organisation+Invite%21+%F0%9F%93%A8&template=please-can-i-join-this-organisation------.md&title=Please+Can+I+Join+This+Organisation%3F+%F0%9F%A5%BA%F0%9F%99%8F") | [Our Website 🌐](https://app-choreography.github.io/) | [Repositories 📚](https://github.com/orgs/App-Choreography/repositories)

------------------------------

@App-Choreography/community
Thanks to EddieHubCommunity, we have a cool **App-Choreography LinkFree profile**
Check it out here: http://linkfree.eddiehub.org/App-Choreography

**Comment by MrSlimCoder** [comment]
@CodingSpecies Amazing
